CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ PROJECT(typelib)
 SET(PROJECT_VERSION 1.1)
 SET(API_VERSION 1)
 
+cmake_policy(SET CMP0007 NEW)
```

Comment:
adding a note why and for what this policy is set enables later generations of hackers to continue hacking ;-)